pandas/tests/io/test_parquet.py

```diff
@@ -486,10 +486,9 @@ def test_datetime_tz(self, fp):
         # doesn't preserve tz
         df = pd.DataFrame({'a': pd.date_range('20130101', periods=3,
                                               tz='US/Eastern')})
-
         # warns on the coercion
         with catch_warnings(record=True):
-            check_round_trip(df, fp, expected=df.astype('datetime64[ns]'))
+            check_round_trip(df, fp)
```

Comment:
You will need to do this dependent on the fastparquet version, as older versions still return non-tz data (or we could skip the test for older versions)

Reply:
or probably better: check how it is done for Arrow (`test_basic` in TestParquetPyarrow), it adds a tz column to the tested dataframe dependent on the version.